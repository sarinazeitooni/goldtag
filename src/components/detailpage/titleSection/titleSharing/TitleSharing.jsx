import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
import {faSchool} from "@fortawesome/free-solid-svg-icons";
import {BrowserView} from "react-device-detect";
import React from "react";
function TitleSharing({id}){
    const socialMedia = [1,2,3];
    // let socialMedia = new Array(3);
    return(
        <BrowserView>
            <div className='sharing-container'>
                <div>{id.share}</div>
                <div>
                    {
                        socialMedia.map((item)=>{
                            return(
                                <span className="social-media-icon"><FontAwesomeIcon icon={faSchool} /></span>
                            )
                        })
                    }
                </div>
            </div>
        </BrowserView>
    )
};
export default TitleSharing;