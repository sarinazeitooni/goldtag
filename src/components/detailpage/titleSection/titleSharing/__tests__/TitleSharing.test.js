import React from 'react';
import { shallow } from 'enzyme';
import TitleSharing from "../TitleSharing";
describe("TitleSharing component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<TitleSharing id={{share : ''}}/>)
        expect(container).toMatchSnapshot();
    });
});