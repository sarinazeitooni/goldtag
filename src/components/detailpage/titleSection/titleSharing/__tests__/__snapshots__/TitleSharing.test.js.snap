// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleSharing component it should render a div 1`] = `ShallowWrapper {}`;
